FRAuthenticator/FRAuthenticator/Policy/BiometricAvailablePolicy.swift

```diff
@@ -0,0 +1,58 @@
+// 
+//  BiometricAvailablePolicy.swift
+//  FRAuthenticator
+//
+//  Copyright (c) 2023 ForgeRock. All rights reserved.
+//
+//  This software may be modified and distributed under the terms
+//  of the MIT license. See the LICENSE file for details.
+//
+
+
+import Foundation
+import LocalAuthentication
+
+/// The Biometric Available policy checks if the device has enabled Biometric capabilities.
+///
+/// JSON Policy format:
+/// {"biometricAvailable": { }}
+public class BiometricAvailablePolicy: FRAPolicy {
+    
+    public var name: String = "biometricAvailable"
+    
+    public var data: Any?
+    
+    public func evaluate() -> Bool {
+        return LAContext().biometricAvailable
+    }
+    
+}
+
+extension LAContext {
+    var biometricAvailable: Bool {
+        var error: NSError?
+
+        guard self.canEvaluatePolicy(.deviceOwnerAuthenticationWithBiometrics, error: &error) else {
+            return biometricResult(result: false)
+        }
+
+        if #available(iOS 11.0, *) {
```

Comment:
Do we need this check? the SDK supports iOS12 and above

Reply:
nice catch!

Reply:
@rodrigoareis seems like this check is still there

Reply:
forgot about that change, done